src/plugin/plugin.ts

```diff
@@ -632,6 +689,184 @@ export interface PurchasesConfiguration {
   useAmazon?: boolean;
 }
 
+/**
+ * Contains all details associated with a SubscriptionOption
+ * Used only for Google
+ */
+export interface SubscriptionOption {
```

Comment:
Do we need to add the `isPrepaid` property here? (Or as discussed [here](https://github.com/RevenueCat/purchases-flutter/pull/721#discussion_r1226184936), in the `StoreProduct`?)

Reply:
Yes we do! I'll get that added 😊